### PR TITLE
CODE RUB: Localize Dependency Exceptions - Contacts

### LINKED ISSUE #530: CODE RUB: Localize Dependency Exceptions - Contacts
Reference: https://github.com/hassanhabib/OtripleS/pull/519

### PR DESCRIPTION
**Pull request by arafat-pro**
Closes #530

### CODE REVIEW

File: OtripleS.Web.Api/Models/Contacts/Exceptions/FailedContactStorageException.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api/Models/Contacts/Exceptions/FailedContactStorageException.cs:1]: Add copyrights header.

File: OtripleS.Web.Api/Models/Contacts/Exceptions/FailedContactStorageException.cs [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+using Xeptions;
+
+namespace OtripleS.Web.Api.Models.Contacts.Exceptions
+{
+    public class FailedContactStorageException : Xeption
+    {
+        public FailedContactStorageException(Exception innerException)
+            :base("Failed contact storage error occurred, contact support.", innerException)
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api/Models/Contacts/Exceptions/FailedContactStorageException.cs:9]: Parameter alias (message: ..)
Run code clean up.

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: OtripleS.Web.Api/Models/Contacts/Exceptions/FailedContactStorageException.cs (hunk 1 above)
Comment (arafat-pro) [reply OtripleS.Web.Api/Models/Contacts/Exceptions/FailedContactStorageException.cs]: thanks for the identification!

File: OtripleS.Web.Api/Models/Contacts/Exceptions/FailedContactStorageException.cs (hunk 2 above)
Comment (arafat-pro) [reply OtripleS.Web.Api/Models/Contacts/Exceptions/FailedContactStorageException.cs]: Parameterized alias as advised!

**Review by ShriHumrudha — APPROVED** [review]

**Pull request merged**